include/megaapi.h

```diff
@@ -9406,6 +9419,9 @@ class MegaApi
          * @return A list of MegaTextChat objects with detailed information about each chatroom.
          */
         MegaTextChatList *getChatList();
+
+        virtual std::string getFileAttribute(MegaHandle handle);
```

Comment:
Please use `char *` or `const char *` if possible for a better compatibility with SWIG.